git/Releasing Process.md

```diff
@@ -0,0 +1,26 @@
+# Releasing process
+
+Let's assume there are only two branches present in repository - `master` and `dev`. Type following commands to make **release**:
+
+1. `git checkout master`
+2. `git pull origin master --rebase`
+3. `git checkout dev`
+4. `git pull origin dev --rebase`
+5. `git checkout -b release dev`
+6. Next step is to update application/project version somewhere (e.g. _config file_, `README.md`, `version.sbt`, etc.). Version is in format `major.minor.patch`. When changes contain new feature then increment `minor` version and set `patch` to **0**. When changes are **bug fixes** then increment `patch` version. Case where `major` is changed should be when we don't support backward compatibility. One of examples would be changing our **API** (_Application Programming Interface_).
```

Comment:
Avoid using word `somewhere`. Write something like _configuration files_.

Reply:
There are given examples for those files.